scripts/globals/beastmentreasure.lua

```diff
@@ -0,0 +1,378 @@
+------------------------------------
+-- Beastmen Treasure side quests
+-- https://ffxiclopedia.fandom.com/wiki/Beastmen_Treasure
+------------------------------------
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require('scripts/globals/weather')
+------------------------------------
+dsp = dsp or {}
+dsp.beastmentreasure = dsp.beastmentreasure or {}
+
+local zoneData = {
+    -- Yuhtunga Jungle (Elshimo Lowlands)
+    [123] =
```

Comment:
Require zones.lua, and use constants for the zones.

```lua
[dsp.zone.YUHTUNGA_JUNGLE]
```

Reply:
Resolved in https://github.com/DarkstarProject/darkstar/pull/6095/commits/d3e56c44ade40cbcb41fe95e8b7ba65acc37f883